yarn.lock

```diff
@@ -13461,19 +13461,17 @@ __metadata:
   version: 0.0.0-use.local
   resolution: "root-workspace-0b6124@workspace:."
   dependencies:
-    "@babel/core": ^7.12.13
-    "@polkadot/api": ^4.0.3
-    "@polkadot/dev": ^0.61.25
+    "@babel/core": ^7.13.10
+    "@polkadot/dev": ^0.62.5
     "@types/jest": ^26.0.20
     bs58: ^4.0.1
     eventemitter3: ^4.0.7
     i18next-scanner: ^3.0.0
     lodash: ^4.17.21
     query-string: ^6.14.1
     sinon-chrome: ^3.0.1
-    store: ^2.0.12
-    stylelint: ^13.9.0
-    stylelint-config-recommended: ^3.0.0
+    stylelint: ^13.12.0
+    stylelint-config-recommended: ^4.0.0
```

Comment:
Do we need to upgrade "@babel/core": ^7.12.13 to ^7.13.10?

If the old versions run fine, let's not change anything on this file, or package.json